Autopilot/PathManager/Src/landingTakeoffManager.cpp

```diff
@@ -0,0 +1,184 @@
+#include "waypointManager.hpp"
```

Comment:
Yeah, looking at your implementations, these classes should not inherit WaypointManager. If you could instead create an instance of waypoint manager in the classes that would be a lot better deign-wise